src/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/SqlProjectResource.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.Build.Evaluation;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Represents a SQL Server Database project resource.
+/// </summary>
+/// <param name="name">Name of the resource.</param>
+public sealed class SqlProjectResource(string name) : Resource(name)
+{
+    internal string GetDacpacPath()
+    {
+        var projectMetadata = Annotations.OfType<IProjectMetadata>().FirstOrDefault();
```

Comment:
I recommend using either `TryGetLastAnnotation` or `TryGetAnnotationsOfType` so that we're using the built-in methods and it would also mean you don't need to do a null check, instead you do a bool test:

```csharp
if (!this.TryGetLastAnnotation<IProjectMetadada>(out var projectMetadata)) {
  throw new InvalidOperationException(...);
}

// do logic when found
```